From: Bjorn Helgaas <bhelgaas@google.com>
Subject: ia64: remove unused __SLOW_DOWN_IO and SLOW_DOWN_IO definitions
Date: Thu, 13 Oct 2022 19:19:10 -0500

Remove unused __SLOW_DOWN_IO and SLOW_DOWN_IO definitions.

Link: https://lkml.kernel.org/r/20221014001911.3342485-3-helgaas@kernel.org
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/ia64/include/asm/io.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/arch/ia64/include/asm/io.h~ia64-remove-unused-__slow_down_io-and-slow_down_io-definitions
+++ a/arch/ia64/include/asm/io.h
@@ -23,10 +23,6 @@
 #include <asm/unaligned.h>
 #include <asm/early_ioremap.h>
 
-/* We don't use IO slowdowns on the ia64, but.. */
-#define __SLOW_DOWN_IO	do { } while (0)
-#define SLOW_DOWN_IO	do { } while (0)
-
 #define __IA64_UNCACHED_OFFSET	RGN_BASE(RGN_UNCACHED)
 
 /*
_
